app/models/miq_template/operations.rb

```diff
@@ -1,4 +1,10 @@
 module MiqTemplate::Operations
+  extend ActiveSupport::Concern
+
+  included do
+    supports :clone
+  end
+
```

Comment:
from the old code, I can see that templates from all providers must support clone operation
`def validate_clone
   {:available => true, :message => nil}
 end`